frontend/src/layout/ErrorProjectUnavailable.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { PageHeader } from 'lib/components/PageHeader'
+
+export function ErrorProjectUnavailable(): JSX.Element {
+    return (
+        <div>
+            <PageHeader title="Project Unavailable" />
+            <p>It seems you don't have access to this project. Switch to a different one.</p>
```

Comment:
We have a nice rocket graphic which we use for 404s, you could use the same graphic here.

Reply:
@paolodamico points lost for not linking to [the relevant storybook](https://storybook.posthog.net/?path=/story/posthog-components-notfound--person&args=object:Project).